src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    void draw(Figure figure);
```

Comment:
When method is called figure draws itself, there is no need to pass some other figure as argument
```suggestion
    void draw();
```